types.d.ts

```diff
@@ -86,8 +86,8 @@ export interface Task extends NodeJS.EventEmitter {
 
     complete(reason: string): Promise<Task>;
     setAttributes(attributes: Object): Promise<Task>;
-    async transfer(to: string, options: TransferOptions): Promise<Task>;
```

Comment:
async is unnecessary and TS doesn't like this in a `d.ts` file. Also see `setAttributes`, `wrapUp`, `transfer`, etc. returns `Promise<Task>` without the redundant `async`.